DEVGUIDE.md

```diff
@@ -123,39 +127,15 @@ For **Release** this corresponds to these steps, which you can run individually
     msbuild src/fsharp-library-unittests-build.proj /p:TargetFramework=portable7 /p:Configuration=Release
     msbuild src/fsharp-library-unittests-build.proj /p:TargetFramework=portable78 /p:Configuration=Release
     msbuild src/fsharp-library-unittests-build.proj /p:TargetFramework=portable259 /p:Configuration=Release
-    msbuild VisualFSharp.sln /p:Configuration=Release
-    msbuild vsintegration\fsharp-vsintegration-unittests-build.proj /p:Configuration=Release
+    msbuild vsintegration/fsharp-vsintegration-src-build.proj /p:Configuration=Release
+    msbuild vsintegration/fsharp-vsintegration-project-templates-build.proj /p:Configuration=Release
+    msbuild vsintegration/fsharp-vsintegration-item-templates-build.proj /p:Configuration=Release
+    msbuild vsintegration/fsharp-vsintegration-deployment-build.proj /p:Configuration=Release
+    msbuild vsintegration\fsharp-vsintegration-unittests-build.proj  /p:Configuration=Release
+    msbuild tests/fsharp/FSharp.Tests.fsproj /p:Configuration=Release
     src\update.cmd release -ngen
     tests\BuildTestTools.cmd release 
 
-### 4. [Optional] Install the Visual F# IDE Tools and Clobber the F# SDK on the machine
```

Comment:
What is the replacement for these instructions?  Does a build now clobber the F# SDK and installed Visual F# IDE Tools on the machine?  If so, for which VS version?